tests/integration/test_lock.py

```diff
@@ -402,64 +401,11 @@ def test_private_index_lock_requirements(PipenvInstance_NoPyPI):
         assert c.returncode == 0
 
 
-@pytest.mark.lock
-@pytest.mark.install
-@pytest.mark.skip_windows
-@pytest.mark.skipif(sys.version_info >= (3, 9), reason="old setuptools doesn't work")
-@pytest.mark.needs_internet
-def test_outdated_setuptools_with_pep517_legacy_build_meta_is_updated(PipenvInstance):
```

Comment:
These tests are the black magic of the devil and old setuptools -- how they ever passed I have no clue and vote we should drop them now.   Just read the comments, it feels like a very old problem it was checking for, and with time we would drop these tests anyway.